### DISCUSSION TITLE
Allow removal of background noise

### DISCUSSION THREAD

**Discussion by Wolfstorm321** [bisq-network/bisq#6971] [General]
There is a very annoying background noise while using BISQ, and I would like a option to disable it. I can't listen mp3, watch videos, etc, with this excessive noise in my ears. I read it had something to do with "wait mode", but after disabling this mode, nothing have changed and the noise is still in my headphones. Only happens while BISQ is open.

**Comment by pazza83** [comment]
The file is called: prevent-app-nap-silent-sound

It is found in the root of the Bisq data directory.

The background for it's inclusion is here: https://github.com/bisq-network/bisq/pull/1712

Not sure how to stop the noise, I do not think most people hear it.

**Comment by ghost** [comment]
Bugfix submitted: https://github.com/bisq-network/bisq/pull/6972
